lib/job-runner.js

```diff
@@ -9,11 +10,26 @@ export const JobStatus = {
 };
 
 export async function run(jobUri, job) {
+  let jobRecord;
   try {
     let jobQuery = Queries.Jobs.buildGet(jobUri);
-    let result = await querySudo(jobQuery);
-    let jobRecord = Queries.Jobs.parseGet(result);
+    // retry mechanism:
+    //  mu-cl-resources does not add attributes and relationships in same INSERT command
```

Comment:
Ik had hierover gehoord tijdens de standup, maar was niet zeker of het ging waarover ik dacht dat het ging. Ik had sneller om verduidelijking moeten vragen. Ik denk dat wat je hier tegenkomt nog eenvoudiger opgelost kan worden door [de delta-notifier `gracePeriod`](https://github.com/kanselarij-vlaanderen/app-kaleidos/blob/a734c9791e037724df1f7cbe97ee4e0143f8ffea/config/delta/rules.js#L108) op te hogen. (zie ook https://github.com/mu-semtech/delta-notifier#receiving-delta-notifications )